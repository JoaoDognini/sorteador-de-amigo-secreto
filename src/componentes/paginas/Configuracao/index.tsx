import Formulario from '../../Formulario'
import ListaParticipantes from '../../ListaParticipantes'
import Rodape from '../../Rodape'
import Card from '../../Card'

export default function Configuracao() {
	return (
		<Card>
			<Formulario />
			<ListaParticipantes />
			<Rodape />
		</Card>
	)
}
 