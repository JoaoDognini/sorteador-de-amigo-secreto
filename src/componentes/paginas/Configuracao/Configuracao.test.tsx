import { RecoilRoot } from "recoil"
import Configuracao from "."
import { render } from "@testing-library/react"

const mockNavigate = jest.fn();
jest.mock('react-router-dom', () => {
	return { useNavigate: () => mockNavigate }
})

describe('A página de configuração', () => {
	test('Deve ser renderizada corretamente', () => {
		const { container } = render(
			<RecoilRoot>
				<Configuracao />
			</RecoilRoot>
		)

		expect(container).toMatchSnapshot();
	})
})