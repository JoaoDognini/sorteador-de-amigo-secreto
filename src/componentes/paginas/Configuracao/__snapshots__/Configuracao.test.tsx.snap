// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A página de configuração Deve ser renderizada corretamente 1`] = `
<div>
  <div
    class="card"
  >
    <section>
      <h2>
        Vamos começar!
      </h2>
      <form
        class="formulario"
      >
        <input
          placeholder="Insira os nomes dos participantes"
          type="text"
          value=""
        />
        <button
          disabled=""
        >
          Adicionar
        </button>
      </form>
      
    </section>
    <ul
      class="lista"
    />
    <footer
      class="rodape"
    >
      <button
        class="rodape_botao"
        disabled=""
      >
        Iniciar
        <br />
        brincadeira!
      </button>
      <img
        alt="Desenho de sacolas de compras"
        src="/imagens/sacolas.png"
      />
    </footer>
  </div>
</div>
`;
